src/control_desk/desk.cpp

```diff
@@ -547,11 +547,13 @@ namespace dmxfish::control_desk {
         col_ptr->set_display_text(msg.lower_display_text(), false);
         if(msg.has_plain_color()) {
             const auto& c = msg.plain_color();
-            col_ptr->set_color(dmxfish::dmx::pixel{c.hue(), c.saturation(), c.intensity()});
+            dmxfish::dmx::pixel new_color = dmxfish::dmx::pixel{c.hue(), c.saturation(), c.intensity()};
+            col_ptr->set_color(new_color);
```

Comment:
i removed a few const, because a getter could change the object :-/
here was a compiler problem, so i introduced a local variable to refer to. is this bad?
oh, i see right now, there is a direct assignment to this color, so i need std::move?

Reply:
No, do not remove them. Make the getter `const` instead. Use it like the following: `RGBColor get_color_as_rgb() const {...}`.

Reply:
did i missunderstand something? then it is not possible that the getter can calculate the r,g,b values and stores them in the object, or am i wrong?